content/rs/administering/cluster-operations/maintenance-mode.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: Maintenance Mode for Cluster Nodes
+description: Prepare node for maintenance, and restore it to previous state
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When you need to do hardware or operating system maintenance on an RS server,
```

Comment:
@bmansheim - Server or node? I think we need to stick to a single term or mention at the beginning the server = node

Reply:
When you need to do hardware or operating system maintenance on a server that hosts an RS node,
it is important that you move all of the shards on that node to another node to avoid the risk of data loss.

Reply:
@Paz-Yanover Do you accept this rewrite?

Reply:
I would prefer not mentioning the negative side of it, but the positive, e.g.-
"it is important that you move all of the shards on that node to another node to protect the data and keep it highly available"- something like that...